### PR TITLE
omnibox: go to arn

### PR DESCRIPTION
**Pull request by brandonbloom**
Implements DEREF-247

### CODE REVIEW

**Comment by brandonbloom** [comment]
Depends on forthcoming `arn2web` from https://github.com/deref/deref-aws

**Comment by brandonbloom** [comment]
Going to do this via the API for a variety of reasons. See https://github.com/deref/deref-graphql/pull/2

**Comment by brandonbloom** [comment]
whoops! obvious stupid bug is obvious. 1 sec

**Comment by brandonbloom** [comment]
ok actually ready now

**Comment by brandonbloom** [comment]
@BenElgar please do a post-hoc review :) thanks

File: lib/omnibox.ts [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import { ExtensionApi } from '~/lib/extension-api/api';
+import { arnToUrl } from '~/lib/arn';
+
+export const initOmnibox = (browser: ExtensionApi) => {
```
Comment (BenElgar) [inline lib/omnibox.ts:4]: Minor but I'd rather keep the `browser` name for the import from the web-extension polyfill. I read the whole function body assuming that was true.

File: dist/manifest.json [hunk 2]
```diff
@@ -37,5 +37,6 @@
     "https://*.auth0.com/",
     "https://*.deref.io/",
     "*://*.aws.amazon.com/*"
-  ]
+  ],
+  "omnibox": { "keyword": "deref" }
```
Comment (BenElgar) [inline dist/manifest.json:41]: Long term I think I'd prefer avoiding the need for a keyword by using the web request API.

File: lib/omnibox.ts (hunk 1 above)
Comment (brandonbloom) [reply lib/omnibox.ts:4]: Sorry, I don't follow. Why? I thought the goal was to mock the `browser` API?

File: dist/manifest.json (hunk 2 above)
Comment (brandonbloom) [reply dist/manifest.json:41]: 👍🏻

**Pull request merged**